pkgdb2/static/pkgdb.css

```diff
@@ -156,6 +156,13 @@ a.wheel, a.wheel:hover, button.wheel, button.wheel:hover {
   background-position: 3px 50%;
 }
 
+label.active, label.active:hover {
+  background-image: url(https://admin.fedoraproject.org/pkgdb/static/arrow.png) !important;
```

Comment:
Yeah, see [here how fedmenu does it?](https://github.com/fedora-infra/fedmenu/blob/develop/css/fedmenu.css#L70)